### PR TITLE
Update main.js

### PR DESCRIPTION
**Pull request by Kadynm49**

### CODE REVIEW

**Review by nydak — DISMISSED** [review]
please don't

**Review by ensayador — APPROVED** [review]

**Review by pengtianyao — COMMENTED** [review]
Test 1

File: main.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const heroku = new Heroku({ token: process.env.HEROKU_API_TOKEN })
 
 const MongoClient = require('mongodb').MongoClient;
 const uri = "mongodb+srv://kadyn:YMZM6XxZAft5QHg@kadyndb.b7mcv.azure.mongodb.net/KadynDB?retryWrites=true&w=majority";
```
Comment (pengtianyao) [inline main.js:11]: test 2

File: main.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const heroku = new Heroku({ token: process.env.HEROKU_API_TOKEN })
 
 const MongoClient = require('mongodb').MongoClient;
 const uri = "mongodb+srv://kadyn:YMZM6XxZAft5QHg@kadyndb.b7mcv.azure.mongodb.net/KadynDB?retryWrites=true&w=majority";
-
+// change it up
 async function checkInStock(url, collection) {
     try {
         const browser = await puppeteer.launch({headless: true, args: ['--no-sandbox']});
```
Comment (pengtianyao) [inline main.js:15]: test 3

**Review by pengtianyao — COMMENTED** [review]
test 4

File: main.js (hunk 2 above)
Comment (pengtianyao) [reply main.js:15]: this

File: main.js [hunk 3]
```diff
@@ -178,4 +178,4 @@ function wait(time) {
 })()
```
Comment (pengtianyao) [inline main.js:178]: is

File: main.js [hunk 4]
```diff
@@ -178,4 +178,4 @@ function wait(time) {
 })()
 
 // https://us.louisvuitton.com/eng-us/products/mini-pochette-accessoires-damier-ebene-001035
```
Comment (pengtianyao) [inline main.js:180]: a

File: main.js [hunk 5]
```diff
@@ -178,4 +178,4 @@ function wait(time) {
 })()
 
 // https://us.louisvuitton.com/eng-us/products/mini-pochette-accessoires-damier-ebene-001035
-// https://us.louisvuitton.com/eng-us/products/voguez-volez-voyagez-paperweight-monogram-metal-nvprod1250322v
```
Comment (pengtianyao) [inline main.js:181]: single

Comment (pengtianyao) [reply main.js:181]: comment

File: main.js [hunk 6]
```diff
@@ -178,4 +178,4 @@ function wait(time) {
...
\ No newline at end of file
+// https://us.louisvuitton.com/eng-us/products/voguez-volez-voyagez-paperweight-monogram-metal-nvprod1250322v
```
Comment (pengtianyao) [inline main.js:181]: to

Comment (pengtianyao) [reply main.js:181]: test

**Pull request merged**